core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -204,7 +204,8 @@ object FileCommitProtocol extends Logging {
       className: String,
       jobId: String,
       outputPath: String,
-      dynamicPartitionOverwrite: Boolean = false): FileCommitProtocol = {
+      dynamicPartitionOverwrite: Boolean = false,
+      staticPartitionInsert: Boolean = false): FileCommitProtocol = {
```

Comment:
hmm, adding one more parameter to the commit protocol API looks a bit overkill to fix this problem, which users can just disable writing the `_SUCCESS` file.

Reply:
> hmm, adding one more parameter to the commit protocol API looks a bit overkill to fix this problem, which users can just disable writing the `_SUCCESS` file.

Update to use flag set in `hadoopConf`